### PR TITLE
Updated if's

### LINKED ISSUE #66: Configure fails due to format of test statement
I am trying to compile an apptainer using debian:bullseye-slim.  The addition of this statement in the configure file stops the compilation

if [[ `uname` -eq Darwin ]] ; then
    compflags=" ${compflags} -isystem /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/v1 -stdlib=libc++ "
fi

as it always equals true. The first test

This format of check works for the bash shells I have tested.

if [[ `uname` == 'Darwin' ]] ; then echo hey ;fi

### PR DESCRIPTION
**Pull request by JustScreaming**
The if statements where failing true under linux bash. Causing the configure to fail as the compiler does not understand the Mac compflags.
resolves #66

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 CXX_STD=CXX17
 JTHREADS=4
-if [[ `uname` -eq Darwin ]] ; then
+if [[ `uname` == 'Darwin' ]] ; then
```
Comment (cookpa) [inline configure:5]: @stnava is there still a reason to only set this on Macs?

**Review by cookpa — COMMENTED** [review]
Looks good but maybe we can remove the first if Darwin altogether

**Comment by stnava** [comment]
this was needed to build on my computer (Apple M2 Max)

File: configure (hunk 1 above)
Comment (cookpa) [reply configure:5]: Yes there is

**Comment by cookpa** [comment]
@JustScreaming I think there was a separate bug in the cxx flags logic (#68).

Because this PR is simple, it might be easier to close and start over - it should build successfully on Linux now

**Comment by cookpa** [comment]
Actually I think I need to fix flags some more, apologies. Will close this and fix

**Pull request closed without merging**